solc/CommandLineInterface.cpp

```diff
@@ -643,6 +644,10 @@ bool CommandLineInterface::parseLibraryOption(string const& _input)
 	{
 		// Thrown e.g. if path is too long.
 	}
+	catch (FileNotFound const&)
+	{
+		// Should not happen if `fs::is_regular_file` is correct.
```

Comment:
Shouldn't `assert(false)` here then?

Reply:
Sounds like a good idea. Or couldn't just not catch the exception and leave a comment as in other places.

However, shouldn't we abort with an error in case of the `PathTooLong` case?

Reply:
Yea think so too

Reply:
Not sure if this should be done here or in a new PR